svc.go

```diff
@@ -162,24 +210,35 @@ func (as *AutoNATService) skipDial(addr ma.Multiaddr) bool {
 		return true
 	}
 
+	// Skip dialing addresses we believe are the local node's
```

Comment:
Is this to prevent an AutoNAT client from getting a false positive if the Server is on the same network ?

Reply:
yep